src/controllers/book.js

```diff
@@ -0,0 +1,12 @@
+const { createItem, getAllItems, getItemById, updateItem, deleteItem } = require('../controllers/helper');
+const bookModel = 'book';
+
+exports.createBook = createItem(bookModel);
+
+exports.getAllBooks = getAllItems(bookModel);
+
+exports.getBookById = getItemById(bookModel);
+
+exports.updateBook = updateItem(bookModel);
+
+exports.deleteBook = deleteItem(bookModel);
```

Comment:
And again with the clear lines. I think you get the point of this now so I wont point out every instance of this and peck your head with it